[SECTION] Agents specific section in hub docs (#2137)
huggingface/hub-docs: docs/hub/agents-cli.md

# Hugging Face CLI for Agents

> Coding agents like Claude Code, OpenAI Codex, and Open Code are excellent at using the CLI to interact with the Hub. The `hf` command-line interface lets you interact with the Hugging Face Hub from your terminal. Search for models, datasets, Spaces, and papers. Download models, upload files, manage repositories, and run compute jobs.

Reviewer: ```suggestion
Coding agents like Claude Code, OpenAI Codex, or Open Code are excellent at using the CLI to interact with the Hub through the `hf` command-line interface. Search for models, datasets, Spaces, and papers. Download models, upload files, manage repositories, and run compute jobs.
```
